src/main/java/entityLayer/CommonUser.java

```diff
@@ -1,4 +1,23 @@
 package entityLayer;
 
 public class CommonUser implements User {
+    @Override
+    public String getEmail() {
+        return null;
+    }
+
+    @Override
+    public Schedule getSchedule() {
+        return null;
+    }
+
+    @Override
+    public void setEmail() {
+
+    }
+
+    @Override
+    public void setPassword() {
+
+    }
 }
```

Comment:
will the user have a username too or just an email?

Reply:
oh also a method for getAvailability

Reply:
I'm looking into adding all of these now; However, I'm gonna hold off on creating a getAvailability method for the user, instead having client code first get the _schedule_, then calling the Schedule's getAvailability method (which, looking now, I'm having to implement. Oops. Guess that's why we do code reviews!)